cmd/server/main.go

```diff
@@ -50,6 +50,7 @@ func main() {
 	e.POST("/nip47", svc.NIP47Handler)
 	e.POST("/nip47/webhook", svc.NIP47WebhookHandler)
 	e.POST("/nip47/notifications", svc.NIP47NotificationHandler)
+	e.POST("/nip47/notifications/go", svc.NIP47ExpoNotificationHandler)
```

Comment:
Should it know about "go"? and this method is to register push notifications, right?

Reply:
Yeah, the URL shouldn't include, let's keep it generic.

Reply:
Changed to `/nip47/notifications/push`